src/index.html

```diff
@@ -45,7 +45,16 @@
           </section>
         </section>
         <section id="main">
-          <h1>main</h1>
+          <section id="buttons">
+            <button>Follow</button>
+            <button>Menu</button>
+          </section>
+          <nav id="tabs">
+            <h3 class="contributions">Contributions</h3>
```

Comment:
Interesting use of `<h3>`! Did our three little things this morning help clarify semantic tags and nav building?